.github/workflows/basic_cli_build_release.yml

```diff
@@ -1,10 +1,15 @@
 on:
+  #pull_request:
```

Comment:
should this be removed?

Reply:
No, I left it there because using the `pull_request` trigger is convenient when the release process requires some debugging.